engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2294,6 +2294,28 @@ private void requestContainer(PTContainer c)
     }
   }
 
+  public void deployAfterRestart()
+  {
+    if (restarted) {
+      LOG.info("App will be deployed from the previous checkpoint.");
```

Comment:
"Redeploying application from the previous checkpoint"? Can you provide more metadata regarding the checkpoint, for example a timestamp?